notebookUtils.py

```diff
@@ -180,7 +180,7 @@ def notebookSetup(c):
 
     # Set a password
     if 'PASSWORD' in os.environ and os.environ['PASSWORD']:
```

Comment:
The ENV name won't change to TOKEN as well?

Reply:
nope, we're going to just use the same one, no real need to change it.

Reply:
Ok